main.go

```diff
@@ -101,23 +104,64 @@ func wrapper(fn func(w http.ResponseWriter, r *http.Request)) http.HandlerFunc {
 func proxy(w http.ResponseWriter, r *http.Request) {
 	params := mux.Vars(r)
 	obj := client.Bucket(params["bucket"]).Object(params["object"])
-	attr, err := obj.Attrs(ctx)
-	if err != nil {
-		handleError(w, err)
-		return
-	}
-	setStrHeader(w, "Content-Type", attr.ContentType)
-	setStrHeader(w, "Content-Language", attr.ContentLanguage)
-	setStrHeader(w, "Cache-Control", attr.CacheControl)
-	setStrHeader(w, "Content-Encoding", attr.ContentEncoding)
-	setStrHeader(w, "Content-Disposition", attr.ContentDisposition)
-	setIntHeader(w, "Content-Length", attr.Size)
-	objr, err := obj.NewReader(ctx)
-	if err != nil {
-		handleError(w, err)
-		return
+	switch r.Method {
+		case http.MethodGet, http.MethodHead:
+			attr, err := obj.Attrs(ctx)
+			if err != nil {
+				handleError(w, err)
+				return
+			}
+			objr, err := obj.NewReader(ctx)
+			if err != nil {
+				handleError(w, err)
+				return
+			}
+			u, err := url.Parse(r.RequestURI)
+			if err != nil {
+				handleError(w, err)
+				return
+			}
+			if u.Query().Get("alt") != "media" {
+				setStrHeader(w, "Content-Type", "application/json")
+				jsonMetadata, err := json.Marshal(attr)
+				if err != nil {
+					handleError(w, err)
+					return
+				}
+				w.Write(jsonMetadata)
```

Comment:
This changes the default behaviour of the upstream gcsproxy - if you perform a GET against an object through the proxy, before it would return the contents of the object; now it returns the metadata in JSON. Can you change this back to the default behaviour and request metadata explicitly please, for example:

```
curl -X GET 127.0.0.1:8080/mybucket/myobject?alt=metadata
```